### PR TITLE
branch protection test workflow

### PR DESCRIPTION
**Pull request by zeelz**
testing branch protection with workflow

### CODE REVIEW

**Comment by zeelz** [comment]
Good to go

**Review by jprodyx — APPROVED** [review]
Good!

**Review by jprodyx — CHANGES_REQUESTED** [review]
Look at L93 for possible improvement

**Review by jprodyx — APPROVED** [review]
Approved

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: CD Pipeline
 
 on:
   workflow_run:
-    workflows: ["Docker Image CI__NOT-FOUND"]
+    workflows: ["Docker Image CI__DISABLE_WKFLW"]
```
Comment (zeelz) [inline .github/workflows/cd.yml:5]: comment 1

**Pull request merged**